certified-connectors/Blackbaud SKY Add-ins/apiDefinition.swagger.json

```diff
@@ -84,6 +84,36 @@
           }
         }
       }
+    },
+    "/virtual/httprequest": {
+      "post": {
+        "operationId": "SendHttpRequest",
+        "summary": "Send an HTTP request",
+        "description": "Sends an HTTP request to the specified SKY API endpoint.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/SendHttpRequestMessage"
+            },
+            "description": "An object that describes the request to be sent to the SKY API.",
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Response",
+            "schema": {}
```

Comment:
@Blackbaud-BenLambert empty schema is not allowed, please define a proper schema.

Reply:
Ok, updated per our discussion - lmk if you need anything else, and thanks again for the help!